android/src/main/java/com/bitmovin/player/reactnative/RNPlayerViewManager.kt

```diff
@@ -99,9 +124,16 @@ class RNPlayerViewManager(private val context: ReactApplicationContext) : Simple
      * @param view Target `RNPlayerView`.
      * @param playerId `Player` instance id inside `PlayerModule`'s registry.
      */
-    private fun attachPlayer(view: RNPlayerView, playerId: NativeId?) {
+    private fun attachPlayer(view: RNPlayerView, playerId: NativeId?, playerConfig: ReadableMap?) {
         Handler(Looper.getMainLooper()).post {
             val player = getPlayerModule()?.getPlayer(playerId)
+            // set picture in picture value based on playback config's option
```

Comment:
NIT: Questionable if this comment is needed. IMO it is not needed.

Reply:
Yeah, you're right. I thought they could help other people see what was my thinking process when working on this feature, but they're actually polluting the code more than helping. So good catch, I'll remove them.

Reply:
Done 8da9939266acb2fc41cd337092888c511d6bcefb